vocabularies/Hierarchy.json

```diff
@@ -86,6 +86,26 @@
         "$Type": "Edm.PropertyPath",
         "$Nullable": true,
         "@Core.Description": "Integer property of type `Edm.Int64` holding the the number of [matching](#MatchCount) descendants a node has in the unlimited hierarchy"
+      },
+      "ChangeNextSiblingAction": {
+        "$Type": "Core.QualifiedActionName",
+        "$Nullable": true,
+        "@Core.Description": "Action that moves a node among its siblings, following [this template](#Template_ChangeNextSiblingAction)"
+      },
+      "ChangePreviousSiblingAction": {
+        "$Type": "Core.QualifiedActionName",
+        "$Nullable": true,
+        "@Core.Description": "Action that moves a node before a newly-created sibling, following [this template](#Template_ChangePreviousSiblingAction)"
+      },
+      "InsertAction": {
+        "$Type": "Core.QualifiedActionName",
+        "$Nullable": true,
+        "@Core.Description": "Action that inserts a new root into a recursive hierarchy, following [this template](#Template_InsertAction)"
```

Comment:
"... inserts an additional root ..." ?

Reply:
"inserts a root", because this can also be the first node at all in the hierarchy.